include/wasmtime.hh

```diff
@@ -35,16 +35,73 @@
 #include <any>
 #include <initializer_list>
 #include <iostream>
+#include <limits>
 #include <memory>
 #include <optional>
-#include <span>
 #include <variant>
 #include <vector>
+#ifdef __cpp_lib_span
```

Comment:
Can you test to make sure that this is exposed with C++20? In my last test I thought that this macro wasn't defined until you included the `<span>` header, but I may not have tested super well.

Reply:
According this table https://en.cppreference.com/w/cpp/feature_test `__cpp_lib_span` is predefined macro in each translation unit

Reply:
And yes this macro is available after `set(CMAKE_CXX_STANDARD 20)`